playbooks/roles/mattermost/tasks/mattermost.yml

```diff
@@ -136,3 +136,17 @@
         state: touch
         owner: mattermost
         group: mattermost
+
+    - name: cleanup old Mattermost installer files from /tmp
+      shell: find /tmp -maxdepth 1 -name 'mattermost-autolink*.tar.gz' | sort | head -n -2
```

Comment:
@AdditionalPylons Can you try to use the built-in Ansible tools like [find](https://docs.ansible.com/ansible/latest/collections/ansible/builtin/find_module.html) along with [file](https://docs.ansible.com/ansible/latest/collections/ansible/builtin/file_module.html)? It will allow the task to be more resilient to changes.

Reply:
@toxinu Please see https://tasks.opencraft.com/browse/BB-3262?focusedCommentId=179148&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-179148 for why I used shell instead of find. Maybe there is a better way to handle it that I'm not aware of? I'm pretty new to writing ansible tasks

Reply:
@0x29a and @toxinu Please take a look at the updated PR. Since this is my first ansible story, I'm not really sure of a good way to test it locally, but I believe this ought to do the trick.